lib/internal/Magento/Framework/GraphQl/Query/QueryComplexityLimiter.php

```diff
@@ -53,7 +61,7 @@ public function __construct(
     public function execute(): void
     {
         DocumentValidator::addRule(new QueryComplexity($this->queryComplexity));
-        DocumentValidator::addRule(new DisableIntrospection());
+        DocumentValidator::addRule(new DisableIntrospection((int) $this->introspectionConfig->isIntrospectionDisabled()));
```

Comment:
@naydav I updated this line casting the bool to int after this change way pushed yesterday https://github.com/magento/graphql-ce/pull/308/commits/b6fe3ff39e221b6604b29753f5c378d93a063296 Passing anything other than `int 0` here disables introspection https://github.com/webonyx/graphql-php/blob/v0.12.6/src/Validator/Rules/DisableIntrospection.php#L31